tests/test_parse_input.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import trailmap.parse_input as pi
+import xml.etree.ElementTree as ET
+
+
+def test_find_commod():
+    tree = ET.parse('input/source_1_sink_1.xml')
+    root = tree.getroot()
+    out_commod_tags = ['commodity']
+    for archetype_var in root.find('./facility/config/Source'):
+        commods = pi.find_commod(archetype_var, 'outcommod')
+
+        assert out_commod_tags == commods
+        break
```

Comment:
why break here, I feel like it will interrupt your `for` loop

Reply:
fixed so it doesn't just look at the first pass through the loop. Thanks for pointing that out!